package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "3d-tiles-validator-1.1",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "description": "Tools for validating 3D Tiles tilesets.",
+  "keywords": [
+    "3D Tiles"
+  ],
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "dependencies": {
+    "ajv": "^8.11.0",
+    "ajv-errors": "^3.0.0",
+    "ajv-formats-draft2019": "^1.6.1",
+    "cesium": "^1.97.0",
+    "gltf-validator": "^2.0.0-dev.3.9",
+    "yargs": "^17.5.1"
+  },
+  "devDependencies": {
+    "@tsconfig/node16": "^1.0.3",
+    "@types/jasmine": "^4.0.3",
+    "eslint": "^8.0.1",
+    "eslint-config-cesium": "^9.0.0",
+    "eslint-config-prettier": "^8.3.0",
+    "jasmine": "^4.3.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.7.4"
+  },
+  "scripts": {
+    "eslint": "eslint \"./**/*.*s\" --cache --quiet",
```

Comment:
`npm run eslint` isn't working out of the box for me

```
(base) slilley@Hydrogen:~/Code/3d-tiles-validator$ npm run eslint

> 3d-tiles-validator-1.1@0.0.0 eslint
> eslint "./**/*.*s" --cache --quiet


Oops! Something went wrong! :(

ESLint: 8.23.1

ESLint couldn't find the plugin "eslint-plugin-node".

(The package "eslint-plugin-node" was not found when loaded as a Node module from the directory "/home/slilley/Code/3d-tiles-validator".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-node@latest --save-dev

The plugin "eslint-plugin-node" was referenced from the config file in ".eslintrc.json » eslint-config-cesium/node".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

Reply:
I have fixed the linting to work for now (even though that involves ignoring the linting for some legacy code parts).

But there are several open questions about the configuration that are addressed below, and that I added as "High Level" discussion points in the implementation notes.